xla/pjrt/c_api_client/pjrt_c_api_client.cc

```diff
@@ -1044,13 +1077,107 @@ PjRtCApiClient::MakeCrossHostReceiveBuffers(
   RETURN_STATUS_IF_PJRT_ERROR(
       extension->PJRT_Transfers_PJRT_Client_MakeCrossHostReceiveBuffers(&args),
       c_api);
-  std::vector<std::unique_ptr<PjRtBuffer>> buffers;
-  buffers.reserve(args.num_buffers);
+
+  return MakePjRtBuffersFromPJRT_Buffers(this, args.buffers,
+                                         temp_buffers.size());
+}
+
+absl::StatusOr<std::vector<Future<>>> PjRtCApiClient::CrossHostSendBuffers(
+    absl::Span<PjRtBuffer* const> buffers,
+    absl::Span<const PjRtGlobalDeviceId> dst_global_device_ids,
+    std::vector<CrossHostTransferKey> transfer_keys) {
+  // Get C API extension.
+  const PJRT_Api* c_api = pjrt_c_api();
+  PJRT_CrossHostTransfers_Extension* extension =
+      FindExtension<PJRT_CrossHostTransfers_Extension>(
+          PJRT_Extension_Type::PJRT_Extension_Type_CrossHostTransfers);
+  if (extension == nullptr) {
+    absl::Status error = absl::UnimplementedError(
+        "CrossHostSendBuffers is not implemented in this PJRT plugin.");
+    std::vector<Future<>> futures;
+    futures.reserve(buffers.size());
+    for (int i = 0; i < buffers.size(); ++i) {
+      futures.push_back(Future<>(error));
+    }
+    return futures;
```

Comment:
Same comment as above, why not simply:

```suggestion
    return absl::UnimplementedError(
        "CrossHostSendBuffers is not implemented in this PJRT plugin.");
```